javascript/javascript1/week1/ageify.js

```diff
@@ -0,0 +1,4 @@
+let yearOfBirth = 1997;
+let yearFuture = 2045;
+const age = yearFuture - yearOfBirth;
+console.log("You will be " + age + " years old in " + yearFuture + ".");
```

Comment:
This works, but I wonder why you have age as const? what if i now want to add:
yearFuture=2050;
and recompute age
age= yearFuture - yearOfBirth;
would this work?

Reply:
Thank you for your feedback ☺️ you are right! I tried changing a variable and the formula didn't work. Do I get right that I should only use "let" with formulas, so it works when a variable is changed?

Reply:
yes. const declarations don't allow variables to be changed afterwards and they should therefore only be used when you know the variable.. erm.. will be a constant ;)  In this case, arguably, the year of birth could be said to be a constant, or you could say they are all modifiable. Most important thing is to understand the difference between the two kinds of declarations because later on you might get in trouble when you have more complex code

Reply:
Fixed the mistakes in the further commits :) A big thank you for explaining it to me!